Message.h

```diff
@@ -465,7 +465,7 @@ class RemoteEventMsg : public Message {
     if( lowbat == true ) {
       flags |= 0x80; // low battery
     }
-    Message::init(0xb,msgcnt,0x40, BIDI|WKMEUP,(ch & 0x3f) | flags,counter);
+    Message::init(11,msgcnt,AS_MESSAGE_REMOTE_EVENT,BIDI|WKMEUP,(ch & 0x3f) | flags,counter);
```

Comment:
Please do not change the length from HEX to DEC.

Reply:
Just for my understanding: Why not? IMHO formatting it as decimal number makes it look like some magic flag value, and additionally it incurs some mental overhead when trying to follow packet layout because one needs to convert hex-to-dec on the fly in that case. This is especially true in cases where the length value is the result of some calculation (e.g. in the peer list message).

Reply:
This makes debugging more easy. No need to "convert" values from console to source code.

Reply:
I do not exactly agree to that idea (adapting source code to conform to a certain debug output format rather than to a human line of thinking, particularly if calculations are involved), but I've updated the commit to match your request.